api/client/samples/analysis_kits/stocks_to_use_price_model/stocks_to_use_price_model.ipynb

```diff
@@ -0,0 +1,590 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Stocks to Use vs Price\n",
+    "\n",
+    "#### *The Python packages used in this notebook are listed in the requirements.txt file located in the current directory. The packages can be installed by simply running 'pip install --requirement requirements.txt' on your command line. Adjustments may be needed if you are not using the same versions included in the requirements file.*\n",
```

Comment:
Will want to pull the installation/overview out into a README file like https://github.com/gro-intelligence/api-client/pull/93/files in the near future, but I think having it in here is good enough for this PR. Can be a follow-up.

Reply:
sounds good